### PR TITLE
added ProjectCard

### PR DESCRIPTION
**Pull request by Jashpatel1**

### CODE REVIEW

File: frontend/src/views/Timeline.vue [hunk 1]
```diff
@@ -6,7 +6,7 @@
 </template>
 
 <script>
-import TimelineCard from "../components/TimelineCard.vue";
+import TimelineCard from "../components/Timelinecard.vue";
```
Comment (Tanuj22) [inline frontend/src/views/Timeline.vue]: Undo this change

**Review by Tanuj22 — CHANGES_REQUESTED** [review]

File: frontend/src/views/Timeline.vue (hunk 1 above)
Comment (Jashpatel1) [reply frontend/src/views/Timeline.vue]: The component name is Timelinecard.
It was written wrongly in Sandesh's PR. So i changed it.

Comment (Tanuj22) [reply frontend/src/views/Timeline.vue]: It shouldn't be. Change that also

Comment (Jashpatel1) [reply frontend/src/views/Timeline.vue]: done
